### ISSUE TITLE
Duplicate system tray icons with NVDA 2021.1beta5

### ISSUE THREAD

**Issue by kvark128** [nvaccess/nvda#12605]
### Steps to reproduce:
1. Run NVDA 2021.1beta5.
2. Press NVDA+N to open the NVDA menu and then close it with the Escape key. Without this step, the issue is not reproduced.
3. Restart NVDA with Control+Alt+N.

### Actual behavior:
There are two NVDA icons in the system tray. Each subsequent restart of NVDA with Control+Alt+N adds another icon.

### Expected behavior:
Only one NVDA icon should be displayed in the system tray.

#### NVDA installed/portable/running from source:
installed

#### NVDA version:
2021.1 beta5

#### Windows version:
Windows 10 version 1809

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
In NVDA 2020.4, the problem is not reproduced.

#### If add-ons are disabled, is your problem still occurring?
Yes

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes
[nvda-old.log](https://github.com/nvaccess/nvda/files/6752318/nvda-old.log)

**Comment by CyrilleB79** [comment]
Cc @seanbudd

**Comment by Brian1Gaff** [comment]
I'm sure this is a very well known weirdness. It also seems very hit and 
miss, however clicking the  old icons will remove them, or at least it seems 
to most of the time.


***@***.***
Sent via blueyonder.
Please address personal E-mail to:-
***@***.***, putting 'Brian Gaff'
in the display name field.

**Comment by CyrilleB79** [comment]
This is actually a regression of NVDA 2021.1beta.

Confirmed on my side with NVDA 2021.1beta5 (replacing the ctrl+alt+N step with Enter on the NVDA desktop shortcut).
And confirmed that the issue is not present on installed NVDA 2020.4.

**Comment by feerrenrut** [comment]
I don't think this is a serious enough issue to prevent the release. We'll aim to fix this in 2021.2
